Sources/TuistKit/Commands/BuildCommand.swift

```diff
@@ -4,6 +4,24 @@ import TSCBasic
 import TSCUtility
 import TuistSupport
 
+enum XcodeBuildPassthroughArgumentError: FatalError, Equatable {
+    case alreadyHandled(String)
+
+    var description: String {
+        switch self {
+        case let .alreadyHandled(argument):
+            "The argument \(argument) added after the terminator (--) cannot be passthrough to xcodebuild because it is handled by tuist"
```

Comment:
```suggestion
            "The argument \(argument) added after the terminator (--) cannot be passed through to xcodebuild because it is handled by Tuist."
```